### PR TITLE
Tickets/dm 34257

### PR DESCRIPTION
**Pull request by pcortes80**
- Modify timeout in **Jenkinsfile.labview**.
- Add Integration Test.
- Update **versionHistory.md**.

### CODE REVIEW

File: doc/versionHistory.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Version History
 
+0.4.1
+
+- Add **integrationTest.vi** in `test/` directory.
```
Comment (teweitsai) [inline doc/versionHistory.md:5]: - Add **integrationTest.vi** in `test/` directory and related tests.

File: doc/versionHistory.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+- Modify **testProcessEvt.vi**, **testSendCmdStatus.vi**, and **testSendEvent.vi** to avoid blocking test when an error is in the prevoius VI.
+
```
Comment (teweitsai) [inline doc/versionHistory.md:7]: Please mention the update of `Jenkinsfile` as well.

**Review by teweitsai — CHANGES_REQUESTED** [review]
It looks like the version of Caraya Unit Test is wrong. Please help to fix it. Thanks!

**Comment by teweitsai** [comment]
[connectTcpClient.vi]

1. The VI description:
Create the connection IDs of TCP clients with TCP server.

[receiveTelemetry.vi]

1. The text in ICON is wrong.

2. The text in label: "because 'this' is the only normal way to stop this loop".

[sendTelFromTcpClient.vi]

1. The VI description:
Then this telemetry --> Then, this telemetry

[testConnectDisconnect.vi]

1. The label "error in (no error)" is far aways from icon in the block diagram.

[testSendCmdFromClient.vi]

1. Do you need the event case of "CmdRead"?

[testSendEvtFromClient.vi]

1. There is a coercion in event case (=2).

2. The remove of 0th and 14th elements is bad. Instead, you can remove the empty string element to be robust.

[testSendEvtToClient.vi]

1. There is a coercion in event case (=2).

2. It looks like one of the while loop can be reused (left-down). I saw this in "sendCmdFromTcpClient.vi" and "receiveTelemetry.vi" and others.

3. The remove of 50th element is bad. Instead, you can remove the empty string element to be robust.

[testSendTelToClient.vi]

1. There is a coercion in event case (=2).

[testSendTelFromClient.vi]

1. There is a coercion in event case (=2).

[testSendTelAndCmdFromClient.vi]

1. There is a coercion in event case (=2).

[testSendTelToClientAndCmdFromClient.vi]

1. There is a coercion in event case (=2).

2. Do you need the event case of "CmdRead"?

3. The text in "define test" is inconsistent with the file name.

File: doc/versionHistory.md (hunk 1 above)
Comment (pcortes80) [reply doc/versionHistory.md:5]: Done

**Comment by pcortes80** [comment]
[connectTcpClient.vi]

The VI description:
Create the connection IDs of TCP clients with TCP server.

Reply: Fixed.

[receiveTelemetry.vi]

The text in ICON is wrong.

The text in label: "because 'this' is the only normal way to stop this loop".

Reply: Fixed.

[sendTelFromTcpClient.vi]

The VI description:
Then this telemetry --> Then, this telemetry

Reply: Fixed.

[testConnectDisconnect.vi]

The label "error in (no error)" is far aways from icon in the block diagram.

Reply: Fixed.

[testSendCmdFromClient.vi]

Do you need the event case of "CmdRead"?

Reply: no, don't need it. Now is removed. Thanks.

[testSendEvtFromClient.vi]

There is a coercion in event case (=2).

Reply: fixed.

The remove of 0th and 14th elements is bad. Instead, you can remove the empty string element to be robust.

Reply: create a vi called "readTcpPackets.vi".

[testSendEvtToClient.vi]

There is a coercion in event case (=2).

Reply: fixed.

It looks like one of the while loop can be reused (left-down). I saw this in "sendCmdFromTcpClient.vi" and "receiveTelemetry.vi" and others.

Reply: create a vi called "readTcpPackets.vi".

The remove of 50th element is bad. Instead, you can remove the empty string element to be robust.

Reply: create a new vi called "removeEmptyStrings.vi"

[testSendTelToClient.vi]

There is a coercion in event case (=2).

Reply: Fixed.

[testSendTelFromClient.vi]

There is a coercion in event case (=2).

Reply: Fixed.

[testSendTelAndCmdFromClient.vi]

There is a coercion in event case (=2).

Reply: Fixed.

[testSendTelToClientAndCmdFromClient.vi]

There is a coercion in event case (=2).

Reply: fixed.

Do you need the event case of "CmdRead"?

Reply: CmdRead event removed.

The text in "define test" is inconsistent with the file name.

Reply: modify "define test" and file name.

**Review by teweitsai — APPROVED** [review]

**Comment by teweitsai** [comment]
Looks good. Thanks for the update!

**Pull request merged**